User Guide 3.2/Tests/test-scoring-and-outcome-declarations.md

```diff
@@ -0,0 +1,50 @@
+
+>For information on Item Scoring Rules, see [Item Scoring Rules](../items/item-scoring-rules.md).
+
+
+Follow the steps below to set up your chosen scoring method.
+
+**1.** After you have created your Test by selecting the Items which are to go in it, hover over the cog wheels just below the Action Bar, at the top of the right-hand corner of the canvas, and you will see the option *Manage Test Properties*. Click on this and the properties relating to the scoring of Tests will open in the right-hand panel. 
+
+
+**2.** Click on the button *Scoring*.
+
+There are four potential configurations from which to choose how you would like the Test to be scored. Enter your choice in *Outcome Processing*:
+
+<!-- Missing Screenshot: Scoring -->
+
+- **None**: If this option is selected, any existing scoring rules will be removed. In this case, no test score will be generated. Scores for the individual items of the Test can be extracted by clicking on *Results*, or the QTI Results API. Use this option if you want to do full custom processing.
+
+- **Custom**: This option is only applicable when a test with custom outcome processing rules is imported - thus the processing rules are defined outside of TAO. In this case, the rules cannot be authored by TAO, so the existing rules are left untouched.
+
+- **Total Score**. If you select this as your chosen configuration, all scores from all parts of the Test will be added together, and the total will be given as the total score. ![Scoring: Total Score](../resources/backend/tests/authoring/settings/test/scoring/total-score.png)
+
+- **Cut Score**: If you choose this option, a cut-off point for passing the test will be fixed. Enter the score which is to be used as the cut-off point in the *Cut Score* box. This is set as a ratio of the total score. A PASS variable is then generated, indicating if the student has passed the test. ![Scoring: Cut Score](../resources/backend/tests/authoring/settings/test/scoring/cut-score.png)
+
+
+If you choose either of the latter configurations (*Total Score* or *Cut Score*), some additional information is required. A check-box for *Category Score* will appear. If *Category Score* is selected, the scores will be calculated for each category separately. You will also need to set the *Weight* identifier used to calculate the score. This should be entered in the *Weight* box. Different *weights* can be assigned to different items of the test, based on how much they should contribute to the final test score. For example, if a weight of 2 is entered, the score for that item is multiplied by 2 when calculating the final test score (see section [Item Scoring Rules](../items/item-scoring-rules.md) for more information on weighting). If any item in the test has no such weight defined on its individual level, the scoring engine assume the default value = 1.
```

Comment:
Maybe we should add or refer to the capability to add categories to item-references? In TAO 3.2, item metadata is automatically transposed into cateogories of the item upon insertion. Categories can also be managed manually.